src/inference/src/dev/iplugin.cpp

```diff
@@ -169,36 +225,193 @@ std::unordered_set<std::string> ov::get_supported_nodes(
+                        } else {
+                            temp_pair_checker[assign->get_variable_id()]++;
+                        }
+                    }
+                    if (ov::op::util::is_constant(op) && !ready_split) {
+                        const auto const_byte_size = op->get_element_type().size() * shape_size(op->get_shape());
+                        total_size += const_byte_size;
+                        // If the total size is 1.05 times larger than the user's requirement:
+                        // - If has_min_graph = false, it means there is no nodes meets requirement, so need cancel
+                        //   split and break
+                        // - If th split_node_set > 1, it means this is not the first search in do-while, so cancel
+                        //   split and break
+                        if (total_size <= max_query_size) {
+                            has_min_graph = true;
+                        } else if (!has_min_graph || search_times > 1) {
+                            cancel_split = true;
+                            break;
+                        }
+                        // Ready to split if total size meets user's requirment and Assign-ReadValue operations in pairs
+                        // on the network
+                        if (total_size >= min_query_size) {
+                            if (!ready_split && split_node_set.find(op->get_friendly_name()) == split_node_set.end()) {
+                                ready_split = check_pairs(temp_pair_checker);
+                                if (ready_split) {
+                                    split_node_set.insert(op->get_friendly_name());
+                                    // Judge if the current constant op should be removed from supported
+                                    if (total_size < max_query_size)
+                                        continue;
+                                }
+                            }
+                        }
+                    }
+                    // Start splitting when ready and the ops is constant
+                    if (ready_split) {
+                        if (ov::op::util::is_constant(op)) {
+                            remove_op_from_supported(op);
+                            start_split = true;
+                        } else if (start_split) {
+                            remove_op_from_supported(op);
+                            for (auto& input : op->inputs()) {
+                                const auto& node = get_input_node(input);
+                                if (ov::op::util::is_constant(node)) {
+                                    remove_op_from_supported(node);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            // In case some ops not in orederd, so traverse the entire model to ensure accurate split
```

Comment:
```suggestion
            // In case some ops not in ordered, so traverse the entire model to ensure accurate split
```

also, can you point me to the case of this situation?

Reply:
Update case and comment, TEST_F(GetSupportedNodesTest, DivideWillRemoveConvertAndConstant)